src/worker/mod.rs

```diff
@@ -2,11 +2,16 @@ use super::proto::{Client, Reconnect};
 use crate::error::Error;
 use crate::proto::{Ack, Fail, Job};
 use fnv::FnvHashMap;
+use std::future::Future;
+use std::pin::Pin;
+use std::process;
 use std::sync::{atomic, Arc};
 use std::{error::Error as StdError, sync::atomic::AtomicUsize};
 use tokio::io::{AsyncBufRead, AsyncWrite};
 use tokio::net::TcpStream;
 use tokio::task::{AbortHandle, JoinSet};
+use tokio::time::sleep as tokio_sleep;
+use tokio::time::Duration as TokioDuration;
```

Comment:
I think this is just a re-export of `std::time::Duration`?

Reply:
Indeed :facepalm: